admin_scripts/os2displaypc/lts_upgrade_in_place_step_1.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+#================================================================
+# HEADER
+#================================================================
+#% SYNOPSIS
+#+    lts_upgrade_in_place_1.sh
+#%
+#% DESCRIPTION
+#%    Step one of the upgrade from 16.04 to 20.04.
+#%
+#================================================================
+#- IMPLEMENTATION
+#-    version         lts_upgrade_in_place_step_1.sh 0.0.1
+#-    author          Carsten Agger
+#-    copyright       Copyright 2020, Magenta Aps
+#-    license         BSD/MIT
+#-    email           info@magenta.dk
+#-
+#================================================================
+#  HISTORY
+#     2021/01/13 : carstena : Script creation
+#
+#================================================================
+# END_OF_HEADER
+#================================================================
+
+# Patch jobmanager to avoid early stoppage.
+sed -i "s/900/900000/" /usr/local/lib/python2.7/dist-packages/bibos_client/jobmanager.py
```

Comment:
I don't really know what to do here. If this line fails and `jobmanager.py` contains this logic, then it's not safe to reboot the computer, but it might also fail because the line isn't there...

I think some better error handling here is in order, anyway.

Reply:
I've tested it many times, but I can check for the return value of `sed` and have the job fail in that case.

Reply:
More info: If the line isn't there, `sed` won't fail, but quietly ignore it (and we don't have a problem). If the line _is_ there, `sed` will fail if it fails to replace the text in question, which is if the file doesn't exist or for some reason we don't have write access  to it.